lpd/models.py

```diff
@@ -236,6 +308,42 @@ def unranked_option_value(cls):
         ).get('number_of_options_to_rank__max')
         return max_rank + 1
 
+    @classmethod
+    def _get_score(cls, value):
+        """
+        Return score corresponding to `value`.
+
+        If learner ranked an answer option belonging to a ranking question,
+        `value` will be equal to the chosen rank.
+
+        If learner did not rank the answer option,
+        `value` will be equal to `RankingQuestion.unranked_option_value`
+        i.e., it will be an arbitrary number that is greater than any of the ranks
+        that the learner can choose from for the given answer option.
+        (Note that this number is computed to be the same across ranking questions
+        to make sure that values scale consistently.)
+
+        To produce a value that will have the desired effect on recommendations
+        generated by the adaptive engine, rescale value x to lie in [0,1]
+        by doing
+
+        x:=(x-m)/(M-m)
```

Comment:
That means that unranked answer options wouldn't be recommended, right? (As they'd have score equal 1, thus to mastery.)

Reply:
@tomaszgy Yes, exactly! That's the behavior we want to trigger in the adaptive engine :)